### PR TITLE
clarify readme regarding installation prerequisites

### PR DESCRIPTION
**Pull request by DenisBiryukov91**
clarify readme regarding installation prerequisites

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,16 @@ To install [zenoh-cpp] do the following steps:
...
 
-   Neither [zenoh-c] nor [zenoh-pico] backends are required for the installation. But at least one of them is required for usage.
+   By default it is expected that you have [zenoh-c] installed. If you want to install for [zenoh-pico] backend or for both (or to not specify any backend), please set `ZENOHCXX_ZENOHC` or `ZENOHCXX_ZENOHPICO` Cmake variables to`ON` or `OFF` accordingly. Notice that at least one of the backends is required for using the library and/or building tests and examples.
 
    Use option `CMAKE_INSTALL_PREFIX` for specifying installation location. Without this parameter installation is performed to default system location `/usr/local` which requires root privileges.
 
     ```bash
     mkdir build && cd build
-    cmake .. -DCMAKE_INSTALL_PREFIX=~/.local
+    cmake ..  -DCMAKE_INSTALL_PREFIX=~/.local # to configure only for zenoh-c backend
```
Comment (sashacmc) [inline README.md:46]:
```suggestion
    cmake .. -DCMAKE_INSTALL_PREFIX=~/.local # to configure only for zenoh-c backend
```
Unnecessary change

**Review by sashacmc — APPROVED** [review]

**Review by milyin — APPROVED** [review]
